src/input.cpp

```diff
@@ -1279,3 +1302,18 @@ void input_context::set_iso( bool mode )
 {
     iso_mode = mode;
 }
+
+std::vector<std::string> input_context::filter_strings_by_phrase( std::vector<std::string> strings,
```

Comment:
Parameters should probably be const references as they are rather complex (-\> reference instead of copy) and never changed by the function (-\> const). Also, the loop here could be written as range-based for loop:
```C++
for( auto &str : strings ) {
    ... // use str instead of strings[i]
```